build-aux/couchdb-build-release.sh

```diff
@@ -18,17 +18,17 @@ CURRENT_BRANCH=`git rev-parse --abbrev-ref HEAD`
 
 # copy sources over
 git archive ${CURRENT_BRANCH} | tar -xC ${REL_DIR}/ -f -
-cd src/
+cd _build/default/lib/
 for repo in *; do
   cd ${repo}
   if [ -d ".git" ]; then
-    mkdir -p ../../${REL_DIR}/src/${repo}
+    mkdir -p ../../../../${REL_DIR}/apps/${repo}
```

Comment:
To improve readability let's set a variable and use that instead.

```
SELF_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
COUCHDB_ROOT="${SELF_DIR}/.."
```

You would need to update `#!/bin/sh -e` in the begining of the script to use bash in order to make it work.